frontend/src/components/Tools/Tools.css

```diff
@@ -0,0 +1,22 @@
+.tools {
+    text-align: center;
+}
+
+.tools-header {
+    margin: 1em;
+    font-weight: 700;
+    font-size: 3em;
+}
+
+.center {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+svg {
```

Comment:
Can you make this a specific class that only affects the svgs' in this component. I fear that if you leave this as is it might leak and affect other svgs' in other components.